### ISSUE TITLE
undestanding use of rewards

### ISSUE THREAD

**Issue by jeweinb** [kzl/decision-transformer#24]
as I am reading through the code I do not understand how you are using rewards to learn an optimal policy. Can you point out where rewards are using in decision transformer during training?

**Comment by jeweinb** [comment]
I see you create a loss_fn in your trainers. But reward is not used here.

loss_fn=lambda s_hat, a_hat, r_hat, s, a, r: torch.mean((a_hat - a)**2),
